docs/_partners/integration-review-guidelines.md

```diff
@@ -65,7 +65,9 @@ The Zapier team verifies each trigger, action,and search in the integration to v
 * Supply any other necessary credentials/resources and configure the test account so it’s ready to test. For example,  providing dummy payment data if your integration requires making payments, or configuring your plugin/embed on a test site to send us. Do not give super admin access or enable any features or settings on the test account that are only available to internal employees, and not available to users.
 
 ### 1.6 Completeness and Platform Consistency
-Submit your integration for review only when it is complete and ready to be published. Make sure it is thoroughly tested for edge cases, and does not have any bugs when submitting. Both your product and integration need to work consistently without errors. See the following guideline regarding “Repeated Submission” for more information. We will not allow incomplete or error-ridden integration integrations to become public.
+Submit your integration for review only when it is complete and ready to be published. Make sure it is thoroughly tested for edge cases, and does not have any bugs when submitting. Both your product and integration need to work consistently without errors. Products not yet fully launched to the public (app is invite-only or for beta users only) and integrations using sandbox/test/dev environments or endpoints will be asked to remain private until they are ready to be publicly available. Once the app is published, it is visible in the Zapier App Directory and it would be expected that any of our mutual users would be able to connect the application to Zapier.
+
+See the following guideline regarding “Repeated Submission” for more information. We will not allow incomplete or error-ridden integration integrations to become public.
```

Comment:
Integration is in this sentence twice, seems like a typo:

`integration integrations`

Reply:
Ooh, thank you!